examples/getting_started/terraform.template.tfvars

```diff
@@ -192,4 +192,18 @@ aws_config = {
   vpc_id     = "vpc-141f0fffff141aa00"
   region     = "us-east-1"
   cidr       = "10.0.0.0/16"
+}
+
+bucket_name     = "test"
+scope_name      = "test"
+collection_name = "test"
+
+index_name   = "idx_pe9"
+index_keys   = ["sourceairport", "destinationairport", "stops", "airline", "id", "ARRAY_COUNT(schedule)"]
+partition_by = ["sourceairport", "destinationairport"]
```

Comment:
The fields seem to be from the travel-sample dataset, but we have mentioned 'test' in the bucket name. So, we are changing the values before running the getting_started folder

Reply:
Bucket name doesn't matter.